pkg/controller/certs/certs.go

```diff
@@ -120,12 +120,12 @@ func CreateSignedServingPair(notAfter time.Time, organization string, ca *KeyPai
 	certDetails := &x509.Certificate{
 		SerialNumber: serial,
 		Subject: pkix.Name{
+			CommonName:   hosts[0],
```

Comment:
Do we do any length checking for the hosts array length anywhere?

Reply:
Right now, it always has exactly two items:

0. `<svcName>.<svcNamespace>`
1. `<svcName>.<svcNamespace>.svc`

Reply:
Alright, works for me.

/lgtm